UniversityApp.BussinessLogic/Configuration/BusinessLogicDI.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using UniversityApp.BussinessLogic.Services;
+using UniversityApp.BussinessLogic.Services.Interfaces;
+
+namespace UniversityApp.BussinessLogic.Configuration
+{
+    public static class BusinessLogicDI
```

Comment:
Maybe better naming would be ServiceCollectionExtensions.cs? This file contains only extension methods